app/src/main/java/com/realityexpander/tasky/agenda_feature/presentation/agenda_screen/AgendaViewModel.kt

```diff
@@ -242,7 +234,7 @@ class AgendaViewModel @Inject constructor(
             }
             is Logout -> {
                 viewModelScope.launch {
-                    workManager.cancelAllWorkByTag(TASKY_WORKERS_TAG)
+                    agendaWorkersScheduler.cancelAllWorkers()
```

Comment:
Is this also called if you're logged out due to an expired or invalid token?

Reply:
Will refactor for this.